make/test/BuildMicrobenchmark.gmk

```diff
@@ -67,8 +67,8 @@ MICROBENCHMARK_NATIVE_EXCLUDE :=
 ###
 
 # Need double \n to get new lines and no trailing spaces
-MICROBENCHMARK_MANIFEST := Build: $(FULL_VERSION)\n\
-\nJMH-Version: $(JMH_VERSION)\n\
+MICROBENCHMARK_MANIFEST := Build: $(FULL_VERSION)\n \
+\nJMH-Version: $(JMH_VERSION)\n \
```

Comment:
Are you sure these spaces are safe? The comment mentions trailing spaces as something undesired and depending on how this is used, I suspect adding a space between the \n could interfere with that.

Reply:
Now that I look at this again, I realize the broken lines start at position 1. That indicates that the original author did not want to indent the line as it "should" have been, just to make sure that no extra space were added. So you are likely correct; this was a bad change. Thanks for spotting it.